tests/admin_widgets/models.py

```diff
@@ -199,3 +199,8 @@ class Profile(models.Model):
 
     def __str__(self):
         return self.user.username
+
+
+class Book(models.Model):
+    title = models.CharField(max_length=255)
+    cover = models.ImageField(upload_to="images", blank=True, null=True)
```

Comment:
This creates the new `images/` directory in my local. How should I manage it to make it temporary?